tests/pos/i4754.scala

```diff
@@ -0,0 +1,12 @@
+object Foo {
+  private final val C = "Constant"
+}
+
+class Foo {
+  import Foo._
+  inline def foo(x: Int): Boolean = x == C.length
+}
+
+class Test {
+  (new Foo).foo(1)
```

Comment:
This should be a run test that checks that no accessor method is present in the class using java reflection.